docs/_articles/en/vscode-desktop/tips-and-tricks.md

```diff
@@ -14,41 +17,41 @@ Organizing your VS Code extensions by development task can help you avoid having
     - Windows
 
       1. When you install VS Code, install `code` as part of your prompt. See [Visual Studio Code on Windows](https://code.visualstudio.com/docs/setup/windows#_installation) in the Visual Studio Code docs for details.
-      1. Open Git Bash. (Git Bash is installed as part of Salesforce CLI.)
-      1. Check whether you have a .bashrc file. If you don’t, create it by running:  
+      2. Open Git Bash. (Git Bash is installed as part of Salesforce CLI.)
+      3. Check whether you have a .bashrc file. If you don’t, create it by running:  
          `touch .bashrc`
-      1. Add this line to your .bashrc file.  
+      4. Add this line to your .bashrc file.  
```

Comment:
I tried to catch where you didn't use ticks for file names. Not sure I caught them all.